crates/adapters/src/util.rs

```diff
@@ -427,27 +427,43 @@ where
 }
 
 struct TokenBucket {
-    available: AtomicU32,
-    last_refill: AtomicU64,
-    suppressed: AtomicU64,
-    first_suppression: AtomicU64,
-    last_suppression: AtomicU64,
-    capacity: u32,
-    ticks_per_token: u64, // in ms
+    /// Number of tokens currently available in the bucket.
+    available_tokens: AtomicU32,
+
+    /// Timestamp (ms since limiter start) when the bucket was last refilled.
```

Comment:
either it's a timestamp, or an interval ("since" implies an interval)
If it's a timestamp, this should say "timestamp when the bucket was last refilled"